webview-ui/src/context/ExtensionStateContext.tsx

```diff
@@ -11,7 +11,7 @@ import { vscode } from "../utils/vscode"
 import { convertTextMateToHljs } from "../utils/textMateToHljs"
 import { findLastIndex } from "../../../src/shared/array"
 
-interface ExtensionStateContextType extends ExtensionState {
+export interface ExtensionStateContextType extends ExtensionState {
```

Comment:
Adding this `export` seems unnecessary because Jest test case was removed in https://github.com/cline/cline/pull/784/commits/22a612636f06303cb935930d10f2111507ad7a57

```suggestion
interface ExtensionStateContextType extends ExtensionState {
```